MoveImage2/IBuilder.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Form
+{
+    interface IBuilder
```

Comment:
Make more specific name as `IBuilder` Is interface for building anything. Be more specific in naming.